dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
I'm not sure. What should we do if the name they supplied for the enrollment differs from the display name on the account? They match by email so the names can diverge.